File: src/cn/langya/sun/value/FloatValue.java
package cn.langya.sun.value;

public class FloatValue {
    private String name;
    private float number;

    public FloatValue(String name, float defaultValue, float max, float min) {
        this.name = name;
        this.number = defaultValue;
    }

    public void set(float value) {
        this.number = value;
    }

    public float get() {
        return this.number;
    }
}

File: src/cn/langya/sun/value/FloatValue.kt
package cn.langya.sun.value


/**
 * @author LangYa
 * @ClassName FloatValue
 * @date 2023/12/15 17:23
 * @Version 1.0
 */

open class FloatValue(name: String,default: Float,max: Float,min: Float){

    var number: Float = 0f

    fun set(float: Float) {
        number = float
    }

    fun get(): Float {
        return number
    }

}